nancy/Dockerfile

```diff
@@ -1,4 +1,10 @@
-FROM golang:1.17.4
+FROM golang:1.19.4
+
+LABEL go_version=1.19.4
+LABEL nancy_version=1.0.29
```

Comment:
i think nancy is now at version:
nancy version 1.0.42

???

Reply:
I don't plan on updating that in this PR.

When I was referring to latest, i ment the tagged docker image for this dockerfile for nancy

Reply:
Ah, yes, i see now its just a label and not an upgrade.